### PR TITLE
Add support for quantitative features

### PR DESCRIPTION
**Pull request by sb-vsfishman**
#### What does this implement/fix? Explain your changes.

This PR and complementary PR from DeepCT allows working with quantitative features (i.e. bigWig format)

#### What testing did you do to verify the changes in this PR?

Tests are available in the DeepCT modules

### CODE REVIEW

File: selene_sdk/targets/q_genomic_features.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+class qGenomicFeatures(Target):
+    """
+    Stores the dataset specifying sequence regions and features.
+    Accepts a path to directory containing bigWig files with feature values
+
+    Parameters
+    ----------
+    features : list(str)
+        non-redundunt fature names
```
Comment (msindeeva) [inline selene_sdk/targets/q_genomic_features.py:13]: typos: non-redundant feature names

File: selene_sdk/targets/q_genomic_features.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    features_path : list(str)
+        locations of coreesponding bigWig files
```
Comment (msindeeva) [inline selene_sdk/targets/q_genomic_features.py:15]: typo: corresponding

File: selene_sdk/targets/q_genomic_features.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    ----------
+    data : tabix.open
+        The data stored in a tabix-indexed `*.bed` file.
+    n_features : int
+        The number of distinct features.
+    """
+
+    def __init__(self, features, features_path):
+        """
+        Constructs a new `GenomicFeatures` object.
```
Comment (msindeeva) [inline selene_sdk/targets/q_genomic_features.py:27]: `qGenomicFeatures`

**Comment by sb-vsfishman** [comment]
Thanks, I've resolved the typos and updated docstrings.

File: selene_sdk/targets/q_genomic_features.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            The 0-based first position in the region.
+        end : int
+            One past the 0-based last position in the region.
+
+        Returns
+        -------
+        numpy.ndarray
+            array of length N, where N is a number of features, and.
+            array[i] is a sum of feature signal over the input genomic
+            interval.
```
Comment (msindeeva) [inline selene_sdk/targets/q_genomic_features.py:58]: typo: "and." -> "and"
Also are you sure it's sum? their doc says it's mean https://github.com/deeptools/pyBigWig#compute-summary-information-on-a-range

Comment (sb-vsfishman) [reply selene_sdk/targets/q_genomic_features.py]: Updated, thanks!

File: selene_sdk/targets/q_genomic_features.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        start : int
+            The 0-based first position in the region.
+        end : int
+            One past the 0-based last position in the region.
+
+        Returns
+        -------
+        numpy.ndarray
+            array of length N, where N is a number of features, and
+            array[i] is a sum of feature signal over the input genomic
```
Comment (msindeeva) [inline selene_sdk/targets/q_genomic_features.py:57]: "mean", not "sum" here as well

**Review by msindeeva — APPROVED** [review]
LGTM

File: selene_sdk/targets/q_genomic_features.py (hunk 5 above)
Comment (sb-vsfishman) [reply selene_sdk/targets/q_genomic_features.py]: Already corrected, pushed it just sec befor your comment. Sorry, no concentration today =)

**Pull request merged**